### DISCUSSION TITLE
What's next to Fritter? -> Turn it to something like a RSS/Feed Reader

### DISCUSSION THREAD

**Discussion by LukasThyWalls** [jonjomckay/fritter#737] [Ideas]
Hello.

Knowing what happened with #699, and understanding it, and also knowing the actual situation with Twitter/Devs/Owner behaviour, i want to ask if @jonjomckay wants to still develop Fritter in some direction, keep and maintain like it is right now with the actual features or abandon it.

Using like it's right now (v3.0.0-beta9) without the feeds, i noticed that how similar is to a News/Feed/RSS Reader. You have the "sources" categorized, you can search to find new "sources", etc.

The only think that lacks is to mark what "sources" are updated since the last time you saw it, i don't know if with the actual situation of the API can be done without fetching the tweets of every profile each time, and if it exist a way it will be a fragile point prone to breakage and changes by Twitter.

I only want to left this here to put my idea.

Thanks for all the work now and in the past.

**Comment by ghost** [comment]
I would love this, may be a bit of reworking required.
